lib/rutabaga/feature.rb

```diff
@@ -3,67 +3,22 @@
 
 module Rutabaga
   module Feature
-    def feature(feature_file = find_feature)
-
+    def feature(feature_file = nil)
+      RSpec.deprecate(
+        "Calling `feature` from an `it` block",
+        :message => "Calling `feature` from an `it` block " \
+                    "is deprecated.\nIt should now be called directly in the " \
+                    "`describe` block."
+      )
+
+      feature_file = Util.find_feature(feature_file || RSpec.current_example.description)
       example_group_class = self.class
 
       # Hack turnip into the rspec only when needed
       example_group_class.send(:include, Turnip::RSpec::Execute)
```

Comment:
I can discuss with you how it works, because it's more complicated than that. And personally, I don't think running steps in isolation is of great value.